src/analysis/seminorm.lean

```diff
@@ -517,6 +517,20 @@ begin
   { rw [finset.sup'_cons hs, finset.inf'_cons hs, ball_sup, inf_eq_inter, ih] },
 end
 
+lemma ball_mono {p : seminorm 𝕜 E} {r₁ r₂ : ℝ} (h : r₁ ≤ r₂) : p.ball x r₁ ⊆ p.ball x r₂ :=
+λ _ (hx : _ < _), hx.trans_le h
+
+lemma ball_antimono {p q : seminorm 𝕜 E} (h : q ≤ p) : p.ball x r ⊆ q.ball x r :=
```

Comment:
```suggestion
lemma ball_antitone {p q : seminorm 𝕜 E} (h : q ≤ p) : p.ball x r ⊆ q.ball x r :=
```

Reply:
I am not happy with that name, but it seems to commonly used in mathlib.